### PR TITLE
Created data model

### PR DESCRIPTION
**Pull request by OpondoG**
## In this Pull request I:

- Created a new blog-app.

- Initialized project with Git.

- Setup Postgres database.

- Builded project schema.

- Created and run the necessary migration files.

- Table and column names matched the ERD diagram.

- Foreign keys included.

- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    t.integer "PostsCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_foreign_key "comments", "posts"
+  add_foreign_key "comments", "users", column: "author_id"
+  add_foreign_key "likes", "posts"
+  add_foreign_key "likes", "users", column: "author_id"
+  add_foreign_key "posts", "users", column: "author_id"
```
Comment (julie-ify) [inline db/schema.rb:60]: - [x] I cannot find any migration files that were used to generate this schema file. It should be noted that schema files should be auto-generated, and updated based on your own migration files. Kindly delete this file and re-generate its contents using migrations.

  image

File: README.md [hunk 2]
```diff
@@ -1 +1,241 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[]**
+- **[]**
+- **[]**
+
```
Comment (julie-ify) [inline README.md]: - [x] Kindly add key features for the app.

File: README.md [hunk 3]
```diff
@@ -1 +1,241 @@
-# BlogApp
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+<img src="https://user-images.githubusercontent.com/99230309/211001724-8b57e176-86a5-46de-995e-71c5fbfef1d7.png" alt="logo" width="140"  height="auto" />
```
Comment (julie-ify) [inline README.md]: - [x] Kindly remove the logo or add yours instead

File: README.md [hunk 4]
```diff
@@ -1 +1,241 @@
...
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
```
Comment (julie-ify) [inline README.md]: - [x] I don't think we are using react and express for this app. Kindly update this part of your readme accordingly

File: README.md [hunk 5]
```diff
@@ -1 +1,241 @@
...
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
```
Comment (julie-ify) [inline README.md]: - [x] Kindly update these sections of your readme file and include necessary steps for setting up and running the app.

File: README.md [hunk 6]
```diff
@@ -1 +1,241 @@
...
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[]**
+- [ ] **[]**
+- [ ] **[]**
+
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly include future features of your app.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @OpondoG,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @OpondoG 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**